### ISSUE TITLE
Multiple Gotify/Matrix bots in same Matrix room?

### ISSUE THREAD

**Issue by damien67** [Ondolin/gotify-matrix-bot#22]
Hi! First of all, thanks a lots for this project! 

I am trying to unifi all notifications from my *arrs container in my Matrix server, which I already use for a lot of different things. Being a little frustrated in the way LSIO's Radarr/Sonarr/Bazarr implement notifications (Radarr needs full blown Apprise server, Sonarr doesn't have Apprise as an option at all, Bazarr just needs an Apprise url), I was looking for a Gotify -> Matrix solution, and came accross yours!

I created 3 users in my matrix server, 3 config volumes for the bots, 3 Gotify apps, and 3 Gotify clients, and attempted to create 3 gotify-matrix-bot containers.

Each config.yaml consists of the following:

gotify:
  url: "my_gotify_url"
  apiToken: "radarr_client_token"
matrix:
  homeserverURL: "my_homeserver_url"
  matrixDomain: "my_homeserver_domain"
  username: "radarr"
  token: "radarr_matrix_token"
  roomID: "my_media_room"
  encrypted: true
debug: false

Where "radarr" above is either radarr/sonarr/bazarr, with their unique Matrix tokens, and with their own Gotify client token.

So in theory, radarr, using its own Gotify app token, notifies Gotify, which notifies gotify-matrix-radarr container, which sends Matrix the message using the radarr token.

Except...I receive a Matrix notification from all bots at the same time!
Testing the notification in Radarr ends up with this in Matrix:

BAZARR
Test Notification
This is a test message from Radarr

SONARR
Test Notification
This is a test message from Radarr

RADARR
Test Notification
This is a test message from Radarr
 
Why is that? I triple checked all configurations and I am certain that all tokens and usernames are different in the config.yaml files.

Any idea?
Thanks!

**Comment by Ondolin** [comment]
Hello,

I'm realy glad my software is helping you even though it is currently not actively maintained or in use by me.

To investigate your issue I'll need some help from you.

1. Can you give me the log files of at least two of the containers?
2. Can you make sure gotify isn't the problem (by taking a look at the messages in gotify using either the android or cli app)

Best regards
Marie

**Comment by damien67** [comment]
Hi Marie, thank you so much for you prompt answer.
In the meantime, I had decided to change the overall setup...maybe I was overcomplicating it a bit.

I simply created a arrsbot user in my Matrix server, created a corresponding Gotify client token, and setup both Radarr and Sonarr to share the same app token. Now I get one notification from both (not trippled!), and all looks good. Bazarr works as expected with the out-of-the-box Apprise integration directly connected to Matrix.

My original issue still remains weird, but may not be worth looking into. Especially if you, creator of this simple bridge, aren't using it!

Again, great work, does exactly what I was looking for!
